content/getting-started/sections/installing-atom.md

```diff
@@ -159,6 +159,41 @@ $ sudo zypper in -y atom.x86_64.rpm
 
 {{/linux}}
 
+#### Updating Atom
+
+You should consider updating Atom periodically for the latest improvements to the software. When Atom receives hotfixes for security vulnerabilities you will want to update your version of Atom as soon as possible. This section will explain how to update Atom on your system.
+
+{{#mac}}
+
+Atom's default setting under the `Atom > Settings` menu will check for updates automatically, but if you don't have "Automatically Update" enabled, you can also update Atom manually.
```

Comment:
Sorry I missed this section in my last review - I think it feels a little ambiguous what setting is referred to at first in "Atom's default setting under the `Atom > Settings` menu"?  The way settings are described in https://flight-manual.atom.io/getting-started/sections/atom-basics/ might be helpful, maybe something like:

```suggestion
"Automatically Update" in Core Settings of the [Settings View](https://flight-manual.atom.io/getting-started/sections/atom-basics/#settings-and-preferences) is enabled by default so Atom will check for updates automatically.  But if you disable this setting, you can update Atom manually.
```